setup.py

```diff
@@ -18,6 +20,34 @@
 from yamllint import linter
 
 
+def find_dirs(base_dir, exclude_dirs, include_dirs, dir_name):
```

Comment:
As of now there is little purpose, but eventually I think we'd want to extend molecule testing to playbooks, and also to segregate tests into more consumable chunks. I can rip this out for now and we can re-introduce something when it's needed.